wagtail/admin/tests/pages/test_edit_page.py

```diff
@@ -1746,7 +1746,17 @@ def test_page_edit_num_queries(self):
         # as when running it within the full test suite
         self.client.get(reverse("wagtailadmin_pages:edit", args=(self.event_page.id,)))
 
-        with self.assertNumQueries(40):
+        with self.assertNumQueries(35):
+            self.client.get(
+                reverse("wagtailadmin_pages:edit", args=(self.event_page.id,))
+            )
+
+        # Check number of queries for non-admin users.
```

Comment:
Suggestion: Checking the number of queries as a different user feels like it should be a separate test case.